app/main.py

```diff
@@ -483,6 +470,17 @@ def get_all_bill_subjects():
             conn.close()
 
 

```

Comment:
```suggestion
@app.route('/health', methods=['GET'])
def health_check():
    return jsonify({'status': 'healthy'}), 200


```
Since we don't have a full swagger ui for the endpoints, we can start here.